examples/polygon/app.tsx

```diff
@@ -53,10 +53,13 @@ function Root() {
       new GeoArrowSolidPolygonLayer({
         id: "geoarrow-polygons",
         data: table,
-        getFillColor: [0, 100, 60, 160],
+        getFillColor: [0, 100, 60, 255],
         pickable: true,
         autoHighlight: true,
-      })
+        workerUrl: new URL(
+          "https://gist.githubusercontent.com/kylebarron/0553b763d118bed6bad48407e9729376/raw/698855d8b237b26af65e2199a4f7914af8e203cf/earcut-worker.js",
```

Comment:
I was getting issues around "failed to construct worker, script cannot be accessed from origin", and wasn't sure if I was doing something wrong on localhost

Reply:
ref https://stackoverflow.com/questions/21913673/execute-web-worker-from-different-origin, https://stackoverflow.com/questions/67326425/construct-worker-from-script-loaded-from-different-origin

Apparently you just can't instantiate a worker cross origin...? That's annoying, but we fetch the worker directly now.